internal/security/setup/option/generate_test.go

```
@@ -172,45 +160,37 @@ func setupGenerateTest(t *testing.T) func(t *testing.T) {
 
 	setup.Init()
```

add _ for the return to indicate ignore the error on the init: 
```go
_ = setup.Init()
```